lifecycle_test.go

```diff
@@ -446,6 +449,19 @@ func sendGpgKeyPair() {
 	coreutils.ExitOnErr(errorutils.CheckResponseStatusWithBody(resp, body, http.StatusCreated))
 }
 
+func importRb(t *testing.T, rbName, rbVersion string) {
+	wd, err := os.Getwd()
+	assert.NoError(t, err)
+	exportedFileName := rbName + "-" + rbVersion + ".zip"
```

Comment:
We can also test the scenario where the exported release bundle has a custom name

Reply:
Custom file name does work, the import process check the content of the file and it has to match, i will add a test for that to be on the safe side 🚀